### ISSUE TITLE
Redis package due for upgrade

### ISSUE THREAD

**Issue by globalexport** [bee-queue/bee-queue#406]
```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ low           │ Regular Expression Denial of Service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ redis                                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=3.1.1                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ bee-queue                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ bee-queue > redis                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://www.npmjs.com/advisories/1662                        │
└───────────────┴──────────────────────────────────────────────────────────────┘
``` 

Low risk of vulnerability, but annoying alerts in CI/CD pipelines.
Are there plans to upgrade to v3?

**Comment by skeggse** [comment]
We're blocked on bee-queue v2, because upgrading to Redis 3 is a breaking change. General support for this project has mostly stalled.

**Comment by skeggse** [comment]
Duplicate of #396.

**Comment by dagli** [comment]
> General support for this project has mostly stalled.

Then can you please update readme?
